src/wordpress-recommendations.lib.php

```diff
@@ -43,103 +43,112 @@ class SucuriWordPressRecommendations
+         * Remove recommendations accordingly.
+         */
+        /*
+         * Check if a SSL cert is being used.
          * @see https://blog.sucuri.net/2019/03/how-to-add-ssl-move-wordpress-from-http-to-https.html
          */
-        if (!is_ssl()) {
-            // phpcs:disable Generic.Files.LineLength
-            $recommendations['noSSL'] = array(
-                __('Implement an SSL Certificate', 'sucuri-scanner') => __('SSL certificates help protect the integrity of the data in transit between the host (web server or firewall) and the client (web browser).', 'sucuri-scanner'),
-            );
-            // phpcs:enable
+        if (is_ssl()) {
+            unset($recommendations['noSSL']);
         }
 
         /*
-         * Check if PHP version being used needs to be upgraded.
+         * Check PHP version.
          * @see https://www.php.net/supported-versions.php
          */
-        if (version_compare(phpversion(), '7.2', '<')) {
-            // phpcs:disable Generic.Files.LineLength
-            $recommendations['PHPVersionCheck'] = array(
-                __('Upgrade PHP to a supported version', 'sucuri-scanner') => __('The PHP version you are using no longer receives security support and could be exposed to unpatched security vulnerabilities.', 'sucuri-scanner'),
-            );
-            // phpcs:enable
+        if (version_compare(phpversion(), '7.2', '>')) {
+            unset($recommendations['PHPVersionCheck']);
         }
 
         /*
-         * Check if WordPress Salt & Security Keys are set and were updated on the last 6 months.
+         * Check if WordPress Salt & Security Keys are set and were updated on the last 12 months.
          * @see https://wordpress.org/support/article/editing-wp-config-php/#security-keys
          * @see https://sucuri.net/guides/wordpress-security/#harrec
          */
-        if (!defined('AUTH_KEY') || !defined('AUTH_SALT')) {
-            // phpcs:disable Generic.Files.LineLength
-            $recommendations['wpSaltExistenceChecker'] = array(
-                __('Missing WordPress Salt & Security Keys', 'sucuri-scanner') => __('Consider using WordPress Salt & Security Keys to add an extra layer of protection to the session cookies and credentials.', 'sucuri-scanner'),
-            );
-        // phpcs:enable
-        } elseif (file_exists(ABSPATH.'/wp-config.php')) {
-            if (filemtime(ABSPATH.'/wp-config.php') < strtotime('-12 months')) {
-                // phpcs:disable Generic.Files.LineLength
-                $recommendations['wpSaltAgeDiscriminator'] = array(
-                    __('WordPress Salt & Security Keys should be updated', 'sucuri-scanner') => __('Updating WordPress Salt & Security Keys after a compromise and on a regular basis, at least once a year, reduces the risks of session hijacking.', 'sucuri-scanner'),
-                );
-                // phpcs:enable
-            }
+        if (defined('AUTH_KEY') || defined('AUTH_SALT')) {
```

Comment:
Previously we showed the warning if one or both weren't defined. Now we show the warning only if both aren't defined. Probably should show it unless both are defined.